<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="PC-02" name="week6.day1.testNgDay3.LeafTapsTestCases" tests="5" failures="0" timestamp="2020-10-23T22:59:22 BST" time="0.028" errors="1">
  <testcase name="editLead" time="0.002" classname="week6.day1.testNgDay3.LeafTapsTestCases"/>
  <system-out/>
  <testcase name="createLead" time="0.003" classname="week6.day1.testNgDay3.LeafTapsTestCases">
    <skipped/>
  </testcase> <!-- createLead -->
  <system-out/>
  <testcase name="createLead" time="0.003" classname="week6.day1.testNgDay3.LeafTapsTestCases">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
at week6.day1.testNgDay3.LeafTapsTestCases.createLead(LeafTapsTestCases.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.TestRunner$$Lambda$50/11288003.accept(Unknown Source)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createLead -->
  <system-out/>
  <testcase name="createLead" time="0.017" classname="week6.day1.testNgDay3.LeafTapsTestCases">
    <skipped/>
  </testcase> <!-- createLead -->
  <system-out/>
  <testcase name="createLead" time="0.003" classname="week6.day1.testNgDay3.LeafTapsTestCases">
    <skipped/>
  </testcase> <!-- createLead -->
  <system-out/>
</testsuite> <!-- week6.day1.testNgDay3.LeafTapsTestCases -->
